app/build.gradle

```diff
@@ -25,7 +25,7 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     compile 'com.android.support:support-v4:23.4.0'
-    compile 'com.google.android.gms:play-services-vision:17.0.2'
+    compile 'com.google.android.gms:play-services-vision:9.4.0+'
```

Comment:
Why did you changed it? 9.4.0 version of library works awful and doesn't recognize a text that the 17.0.2 does.

Reply:
Reverted.